shimmingtoolbox/unwrap/prelude.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/python3
+# -*- coding: utf-8 -*-
+"""prelude is a wrapper function to FSL Prelude
+
+
+"""
+
+import numpy as np
+import os
+import nibabel as nib
+import subprocess
+
+
+def prelude(complex_array, affine, mask=np.array([-1]), path_2_unwrapped_phase='./unwrapped_phase.nii',
```

Comment:
i would input phase and optionally mag, and have the input dims as they are expected by the actual FSL prelude.

if we need to loop across time or echoes, we can use python decorators